proto-public/pbmesh/v2beta1/http_route.proto

```diff
@@ -214,6 +214,8 @@ message HTTPRouteFilter {
   // URLRewrite defines a schema for a filter that modifies a request during
   // forwarding.
   HTTPURLRewriteFilter url_rewrite = 5;
```

Comment:
You don't need to touch this as the v2 code is scheduled for deletion (unless you need to for the xdsv2 compat shim tests to continue to pass in the interim).